maven/plugins/com.liferay.ide.maven.core/src/com/liferay/ide/maven/core/LiferayMavenProjectProvider.java

```diff
@@ -723,7 +723,7 @@ private void updateDtdVersion( IProject project, String dtdVersion, String arche
 
             for( IFile file : metaFiles )
             {
-                editModel = StructuredModelManager.getModelManager().getModelForEdit( file );
+                editModel = StructuredModelManager.getModelManager().getModelForEdit( file ).reinit();
```

Comment:
This shouldn't be necessary if we were properly closing the dom model before.  This is just masking the problem I believe.  Can you double check to make sure that whenever we open the liferay-portlet.xml model we are being careful to close it?